spec/cucumber/multiline_argument/data_table_spec.rb

```diff
@@ -254,10 +254,10 @@ module MultilineArgument
 
       describe '#map_headers' do
         let(:table) do
-           DataTable.from([
-          %w{HELLO WORLD},
-          %w{4444 55555}
-          ])
+          DataTable.from([
```

Comment:
these feel off? Think this change needs undoing (1 space here?)

Reply:
Yeah, there's a couple that somehow ended up funky.